### DISCUSSION TITLE
How to style the search results

### DISCUSSION THREAD

**Discussion by gamue** [mmistakes/minimal-mistakes#2589] [Q&A]
Hey,
first of all thank you very much for creating and maintaining this awesome theme.

One of the next features I want to enable in my blog is the search functionality (starting with lunr first), however I don't like the way it's currently displayed as the images are way too big and I usually show them in a 1:1-style. So I wanted to adjust the layout of the search results, but can't find the code for it or documentation about it.
My goal is to have the results the same way I display the blog posts, see www.gamue.de

Thanks in advance,
Johannes

**Comment by mmistakes** [comment]
Lunr search results uses the same HTML/CSS as other archive listings. Whatever CSS is applied there should apply to the search results unless you changed the markup or class names.

If you changed the markup you'll need to update the search view to match. That can be found here:
https://github.com/mmistakes/minimal-mistakes/blob/master/assets/js/lunr/lunr-en.js#L47-L68

**Comment by gamue** [comment]
Thanks, that script was exactly what I was looking for as I changed the markup.

As it's lunr specific and I want to try algolia in the future, does it work similar or where can I find the markup code for that?
I'm also wondering why my pages aren't showing up in the search results, even I set the following as default in the `_config.yml`

```
  - scope:
      path: _pages/countries
      type: pages
    values:
      classes: galleryPage
      comments: true
      search: true
```

**Comment by mmistakes** [comment]
Only collection documents are looped into the Lunr index. You'd need to modify this file to loop over `site.pages` too.
https://github.com/mmistakes/minimal-mistakes/blob/master/assets/js/lunr/lunr-store.js

The search markup for Algolia can be found here:
https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/search/algolia-search-scripts.html#L26-L31
